tests/phpunit/tests/test-create-delete-languages.php

```diff
@@ -147,6 +147,7 @@ public function test_language_properties_in_transient() {
 			'slug',
 			'term_group',
 			'term_taxonomy_id',
+			'term_props',
```

Comment:
A per our discussion, old term properties should be removed.
However, maybe we could extend the test and check what we have in term_props.

Reply:
Good idea! Done in 957b0b2